compiler/src/iree/compiler/Modules/Check/test/ops.mlir

```diff
@@ -120,7 +120,17 @@ util.func public @expect_almost_eq_const(%lhs : tensor<2x2xf32>) {
 // CHECK-LABEL: @expect_almost_eq_const
 // CHECK-SAME: %[[LHS:[a-zA-Z0-9$._-]+]]
 util.func public @expect_almost_eq_const(%lhs : tensor<2x2xf32>) {
-  // CHECK: check.expect_almost_eq_const(%[[LHS]], dense<1.000000e+00> : tensor<2x2xf32>, tolerance 1.23{{0*}}e+02) : tensor<2x2xf32>
-  check.expect_almost_eq_const(%lhs, dense<1.0> : tensor<2x2xf32>, tolerance 123.0) : tensor<2x2xf32>
+  // CHECK: check.expect_almost_eq_const(%[[LHS]], dense<1.000000e+00> : tensor<2x2xf32>, atol 1.23{{0*}}e+02) : tensor<2x2xf32>
+  check.expect_almost_eq_const(%lhs, dense<1.0> : tensor<2x2xf32>, atol 123.0) : tensor<2x2xf32>
+  util.return
+}
+
+// -----
+
+// CHECK-LABEL: @expect_almost_eq_const
+// CHECK-SAME: %[[LHS:[a-zA-Z0-9$._-]+]]
+util.func public @expect_almost_eq_const(%lhs : tensor<2x2xf32>) {
+  // CHECK: check.expect_almost_eq_const(%[[LHS]], dense<1.000000e+00> : tensor<2x2xf32>, atol 2.5{{0*}}e-01, rtol 1.25{{0*}}e-01) : tensor<2x2xf32>
+  check.expect_almost_eq_const(%lhs, dense<1.0> : tensor<2x2xf32>, atol 0.25, rtol 0.125) : tensor<2x2xf32>
```

Comment:
Can you add a test with only rtol?

Reply:
That actually becomes no longer supported with your other (good) suggestion to drop the custom parser.

Reply:
If you loosen the requirement for `,`s, you can do `oilist`

Reply:
ok sounds good then, marking as resolved.

Reply:
GitHub comments are confusing. Pasting back into this comment thread what I wrote outside of it:

Would like to keep it as-is. I don't feel a strong need for rtol without atol; in fact, this might be an anti-pattern: such tests would suddenly fail if the lhs value happens to be exactly 0.

Reply:
Yeah I'm not attached to such cases, I was just under the impression from the parsing impl that it was supposed to work (so no strong opinion either way from me).